ClosedXML/Excel/Ranges/XLRangeBase.cs

```diff
@@ -2067,5 +2067,38 @@ public void Select()
         {
             Worksheet.SelectedRanges.Add(AsRange());
         }
+
+        /// <summary>
+        /// Get cropped range.
+        /// </summary>
+        public IXLRangeBase Crop(IXLRangeBase crop)
+        {
```

Comment:
What happens if the 2 ranges don't overlap? Do we need error handlers for that case?